keycloak.yaml

```diff
@@ -10,22 +10,22 @@ package:
       license: Apache-2.0
   dependencies:
     runtime:
-      - bash # some helper scripts use bash
+      - bash # Keycloak helper scripts require bash, aren't compatible with busybox.
       - openjdk-17-default-jvm
 
 environment:
   contents:
     packages:
+      - bash
       - busybox
       - ca-certificates-bundle
-      - curl
-      - maven
-      - openjdk-17
-      - openjdk-17-default-jvm
-      - bash
       - nodejs-18
```

Comment:
Similar to the comment above, I couldn't see in the CI build logs where a node build was happening, so just checking that this is needed?

Reply:
Sure, so if node is missing the build will eventually with an error similar to the below:

```
The parameters 'nodeVersion' for goal com.github.eirslett:frontend-maven-plugin:1.12.1:install-node-and-pnpm are missing or invalid
```